sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/XmlPlugin.java

```diff
@@ -56,7 +56,7 @@ public List getExtensions() {
 
       // Sensors
       XmlSensor.class,
-      LineCountSensor.class,
+      LineCounter.class,
```

Comment:
@lindamartin Ok, i see, it's used in XmlSensor. Then it's useless to put it here